src/components/BottomInputComponent.js

```diff
@@ -132,16 +132,16 @@ function BottomInputComponent({
               className='ms-3 bg-info rounded btn-sound-toogle'
             >
               {sound ? (
-                <>
-                  <TbMusicOff title='Sound off' className='text-primary' />
-                </>
-              ) : (
                 <>
                   <BsMusicNoteBeamed
                     title='Sound on'
                     className='text-primary'
                   />
                 </>
+              ) : (
+                <>
+                  <TbMusicOff title='Sound off' className='text-primary' />
```

Comment:
```suggestion
                  <TbMusicOff title='Sound on' className='text-primary' />
```
and we also need to change the other that to be `sound off`